mem0/configs/prompts.py

```diff
@@ -229,5 +229,5 @@ def get_update_memory_messages(retrieved_old_memory_dict, response_content):
     - If there is a deletion, the memory key-value pair should be removed from the memory.
     - If there is an update, the ID key should remain the same and only the value needs to be updated.
 
-    Do not return anything except the JSON format.
+    Only return the json format, do not add ``` around the json response. Start directly with {{ so I can load the json response with json.loads() function. 
```

Comment:
Has this prompt change been tested throughly?

Reply:
This was a change from my other pr - #1884

I don't know how it came here, but yes it has been working with 2 models.

Reply:
Thanks